### DISCUSSION TITLE
Welcome to BlazorRetaurant Discussions!

### DISCUSSION THREAD

**Discussion by efonsecab** [efonsecab/BlazorRestaurant#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alarak666** [comment]
Hello, I'm new to ASP.Net and came across your project, when I run it, unfortunately I get an error
System.ArgumentNullException: 'Value cannot be null. Arg_ParamName_Name'
in CreateHostBuilder
Maybe I didn't write the right appsettings or do I need something else?

**Reply by efonsecab** [reply]
Hello @Alarak666 , please share a screenshot along with the Stack Trace
